src/algebra/module/linear_map.lean

```diff
@@ -564,17 +664,24 @@ by { ext x, simp }
   (linear_equiv.refl R M : M →ₗ[R] M) = linear_map.id :=
 rfl
 
+instance : ring_hom_inv_pair (ring_hom.id R) (ring_hom.id R) := by apply_instance
```

Comment:
Why do you need to register this instance if it is just proved by `apply_instance`?

Reply:
I think this was a test line that we somehow forgot to remove.